<launch>
    <node pkg="prometheus_detection" type="yolov5_tensorrt_client.py" name="yolov5_tensorrt_client" output="screen">
        <param name="output_topic" value="/prometheus/object_detection/yolov5_tensorrt_det"/>
        <param name="camera_parameters" type="string" value="$(find prometheus_detection)/shell/calib_webcam_640x480.yaml" />
        <param name="object_names_txt" value="coco"/>
    </node>
</launch>
